angular-ngrx-scss/src/app/home/profile/profile-about/profile-about.component.html

```diff
@@ -60,8 +60,8 @@ <h1>
       <h2>Organizations</h2>
       <ul>
         <li *ngFor="let org of profile?.orgs">
-          <a href="/orgs/{{ org.login }}">
-            <img [src]="org.avatar_url" />
+          <a routerLink="/{{ org.login }}">
+            <img [src]="org.avatar_url" alt="organization logo" />
```

Comment:
Say the organisation is "thisdot", is "org.login" === "thisdot"?
if yes, we can make the alt text look something like 
``` <img [src]="org.avatar_url" alt="{{org.login}} logo" />```
if there is a null possibility (I doubt), then maybe 
``` <img [src]="org.avatar_url" alt="{{org.login ?? 'organization'}} logo" />```